modules/bitgo/test/unit/bitgo.ts

```diff
@@ -52,7 +52,7 @@ describe('BitGo Prototype Methods', function () {
     it('staging', () => {
       BitGoJS.setNetwork('testnet');
       TestBitGo.decorate(BitGo, { env: 'staging' });
-      BitGoJS.getNetwork().should.equal('bitcoin');
+      BitGoJS.getNetwork().should.equal('testnet');
```

Comment:
Is the test network testnet now instead of bitcoin?

Reply:
for the staging environment, the test network is going to be testnet. we werent really using the staging environment before. but now we have provisioned a fully operational staging environment which is quite close functionally to the test environment. hence the update to the unit test